bing-wallpaper.sh

```diff
@@ -108,3 +112,11 @@ for p in "${urls[@]}"; do
         print_message "Skipping: $filename..."
     fi
 done
+
+if [ $SET_WALLPAPER ]; then
+/usr/bin/osascript<<END
```

Comment:
Could you indent this for better readability?

Reply:
Sorry I just forgot about it and it got off mind. Thanks for merging it BTW. It's my first contribution to open souce :)